tests_zemu/tests/protobuf.test.ts

```diff
@@ -63,10 +63,6 @@ const PB_TXNS = [
     name: 'pb_spawn_neuron',
     blob: 'd9d9f7a167636f6e74656e74a6636172674f620b10f5c88584a9f98ded910122006b63616e69737465725f69644a000000000000000101016e696e67726573735f6578706972791b169bc108342f99006b6d6574686f645f6e616d65706d616e6167655f6e6575726f6e5f70626c726571756573745f747970656463616c6c6673656e646572581d19aa3d42c048dd7d14f0cfa0df69a1c1381780f6e9a137abaa6a82e302',
   },
```

Comment:
Can we keep the stake maturity test but expecting the rejection from the app? Maybe in a different file, so we can add later all the deprecated functions/transactions